manifests/00-ingress-credentials-request.yaml

```diff
@@ -117,7 +125,11 @@ apiVersion: cloudcredential.openshift.io/v1
 kind: CredentialsRequest
 metadata:
   name: openshift-ingress-alibabacloud
-  namespace: openshift-ingress-operator
+  namespace: openshift-cloud-credential-operator
+  annotations:
+    include.release.openshift.io/ibm-cloud-managed: "true"
+    include.release.openshift.io/self-managed-high-availability: "true"
```

Comment:
Talking this over with @joelddiaz , we probably want to keep these out of this profile, to make it more clear to cluster admins that the cloud-cred operator will completely ignore CredentialsRequests on these platforms, regardless of whether it's configured for manual or mint or whatever mode.

Reply:
this does imply we should do this for ALL CredentialsRequests for platforms that are only supported in "Manual" mode...